pkg/server/server_test.go

```diff
@@ -1625,3 +1625,187 @@ func TestWriteAuthorizationModelWithExperimentalEnableModularModels(t *testing.T
 		require.NoError(t, err)
 	})
 }
+
+func TestListUsersValidation(t *testing.T) {
+	t.Cleanup(func() {
+		goleak.VerifyNone(t)
+	})
+
+	model := `
+	model
+		schema 1.1
+	type user
+
+	type document
+		relations
+			define viewer: [user]`
+
+	tests := []struct {
+		name             string
+		req              *openfgav1.ListUsersRequest
+		model            string
+		tuples           []*openfgav1.TupleKey
+		expectedUsers    []string
+		expectedErrorMsg string
```

Comment:
Instead of testing the error message, can we test the grpc status code that the server returns? That's more important, because the error code is subject to change.

Or can we at least add the grpc status code check to the test assertion?